system/Model.php

```diff
@@ -783,11 +783,11 @@ public function update($id = null, $data = null): bool
      * @param object|string $data
      * @param bool          $recursive If true, inner entities will be cast as array as well
      *
-     * @return array|null Array
+     * @return array Array
```

Comment:
```suggestion
     * @return array
```

Reply:
I think this will now be removed by php-cs-fixer

Reply:
Added the description.